lib/modules/subredditInfo.js

```diff
@@ -61,7 +61,8 @@ modules['subredditInfo'] = {
 	srRe: /\/r\/(\w+)(?:\/(new|rising|controversial|top))?\/?$/i,
 	addListeners: function(ele) {
 		ele = ele || document.body;
-		var subredditLinks = document.body.querySelectorAll('.listing-page a.subreddit, .explore-page a.subreddit, .comment .md a[href^="/r/"], .side a[href^="/r/"], .usertext-body a[href^="/r/"], .trending-subreddits a[href^="/r/"]');
+        var querySelectors = ['.listing-page a.subreddit','.explore-page a.subreddit','.comment .md a[href^="/r/"]','.comment .md a[href*="reddit.com/r/"]','.side a[href^="/r/"]','.side a[href*="reddit.com/r/"]','.usertext-body a[href^="/r/"]','.usertext-body a[href*="reddit.com/r/"]','.trending-subreddits a[href^="/r/"]','.trending-subreddits a[href*="reddit.com/r/"]'];
```

Comment:
I wonder if we should perhaps perftest this... it's getting kind of bananas with this number of querySelectors.

might it be faster to querySelector all links and scan them for a subreddit link? I'm honestly not sure, I'm up late / can't sleep and just postulating, but adding this many queryselectors is probably pretty costly.